packages/component/src/Activity2/Timestamp.js

```diff
@@ -0,0 +1,83 @@
+import { connect } from 'react-redux';
+import React from 'react';
+
+import String, { getString } from '../Localization/String';
+import Context from '../Context';
+import TimeAgo from '../Utils/TimeAgo';
+
+// TODO: We could refactor this into a general component
+function sendFailed(language, replace) {
+  const text = getString('Send failed, {retry}', language);
```

Comment:
getstring by key instead of the string? Like resx resjson

Reply:
Our string set is very small now (10-20 strings). Using the English text as ID make it easier to translate for now.